src/modules/dpos/validator/components/delegateProfile/delegateProfile.css

```diff
@@ -45,6 +57,38 @@
     & button {
       grid-column: 1/5;
       margin-top: -15px;
+      border-top: none;
+    }
+  }
+
+  & .accountSummary {
+    display: flex;
+    flex-direction: column;
+    margin-left: 8px;
+
+    & p {
+      @mixin headingSmall;
+
+      --content-font: 'gilroy';
```

Comment:
@mixin headingSmall already sets the font family to gilroy. It also defines a fall back. Please remove this.
Also, if you want to define font weight, you can pass it as
```
@mixin headingSmall bold;
```
Therefore please remove, font family, font weight, font size, and line height.
Also apply this for other CSS rules throughout your PR.